Documentation/fault-output.rst

```diff
@@ -1,36 +1,41 @@
 .. _fault_output:
 
-Fault output
+Fault output - Dynamic Rupture Visualization Options
 ============
 
 Introduction
 ------------
 
-Two types of outputs are available for imaging the rupture. The rupture
-characteristics can be assessed at a set of locations, using ASCII
-receiver files, or overall the whole fault using files that can be
-opened in ParaView. Threads or nodes can be dedicated to write this
-latter output (see :ref:`asynchronous-output`),
-but it is usually not necessary. The type of output generated depends on the value
-of the variable `OutputPointType` of the DynamicRupture namelist:
+There are two primary methods for visualizing on-fault rupture dynamics:
+1. Evaluation of rupture characteristics at specific on-fault locations via ASCII receiver files
+2. Visualization across the entire fault surface with files that can be opened with ParaView
```

Comment:
This is how it renders
![image](https://github.com/SeisSol/SeisSol/assets/13536910/acc679f8-b78e-4742-aab0-f5a4d3475360)

(you can open the file _build/html/fault-output.html after running `make html` in the Documentation folder).
You need to add an empty line before the line starting with 1.